recipes/_configure.rb

```diff
@@ -78,93 +79,203 @@
 
-execute "sentry DB upgrade" do
-  command "#{node["sentry"]["install_dir"]}/bin/sentry --config=#{node["sentry"]["config_file_path"]} upgrade --noinput"
-  user sentry_user
-  group sentry_group
-  action :run
-end
+    variables({
+      db_engine: node["sentry"]["config"]["db_engine"],
+      db_name: sentry_config["database_name"],
+      db_user: sentry_config["database_user"],
+      db_password: sentry_config["database_password"],
+      db_host: sentry_config["database_host"],
+      db_port: sentry_config["database_port"],
+      db_options: node["sentry"]["config"]["db_options"],
+      admin_email: node["sentry"]["config"]["admin_email"],
+      signing_token: sentry_config["signing_token"],
+      public: node["sentry"]["config"]["public"],
+      allow_registration: node["sentry"]["config"]["allow_registration"],
+      beacon: node["sentry"]["config"]["beacon"],
+      url_prefix: node["sentry"]["config"]["url_prefix"].sub(/(\/)+\z/, ""),
+      web_host: node["sentry"]["config"]["web_host"],
+      web_port: node["sentry"]["config"]["web_port"],
+      web_options: node["sentry"]["config"]["web_options"],
+      secure_proxy_ssl_header: node["sentry"]["config"]["secure_proxy_ssl_header"],
+      email_default_from: node["sentry"]["config"]["email_default_from"],
+      email_backend: node["sentry"]["config"]["email_backend"],
+      email_host: node["sentry"]["config"]["email_host"],
+      email_port: node["sentry"]["config"]["email_port"],
+      email_user: sentry_config["email_host_user"],
+      email_password: sentry_config["email_host_password"],
+      email_use_tls: node["sentry"]["config"]["email_use_tls"],
+      email_subject_prefix: node["sentry"]["config"]["email_subject_prefix"],
+      additional_apps: Array(node["sentry"]["config"]["additional_apps"]),
+      prepend_middleware_classes: Array(node["sentry"]["config"]["prepend_middleware_classes"]),
+      append_middleware_classes: Array(node["sentry"]["config"]["append_middleware_classes"]),
+      redis_enabled: node["sentry"]["config"]["redis_enabled"],
+      redis_config: node["sentry"]["config"]["redis_config"],
+      cache: node["sentry"]["config"]["cache"],
+      celery_always_eager: node["sentry"]["config"]["celery_always_eager"],
+      broker_url: node["sentry"]["config"]["broker_url"],
+      celeryd_concurrency: node["sentry"]["config"]["celeryd_concurrency"],
+      celery_send_events: node["sentry"]["config"]["celery_send_events"],
+      celerybeat_schedule_filename: node["sentry"]["config"]["celerybeat_schedule_filename"],
+      ratelimiter: node["sentry"]["config"]["ratelimiter"],
+      buffer: node["sentry"]["config"]["buffer"],
+      quotas: node["sentry"]["config"]["quotas"],
+      tsdb: node["sentry"]["config"]["tsdb"],
+      filestore: node["sentry"]["config"]["filestore"],
+      filestore_options: node["sentry"]["config"]["filestore_options"],
+      # google sso
+      google_client_id: node['sentry']['google_client_id'],
```

Comment:
Is this an attribute or a data bag item?
It looks like an attribute. If so, I'd declare this in the attributes file with a default falsey value.